person.rb

```diff
@@ -22,6 +25,10 @@ def correct_name
     @name
   end
 
+  def add_rental(book, date)
+    @rentals << Rental.new(book, date, self)
+  end
```

Comment:
- [ ] Remember that in the Rental class in lines 9 and 10 the rental adds itself to the arrays of book and person objects so we don't need to duplicate that here.

- [ ] We should also ensure we follow the order of the arguments as outlined in the rental constructor: date, book, person

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```